mediacapture-streams/enumerating-local-media-devices/mediadevices/api-present.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html>
+<head>
+<title>getUserMedia: test that getUserMedia is present (with or without vendor prefix)</title>
+<link rel="author" title="Dr Alex Gouaillard" href="mailto:agouaillard@gmail.com"/>
+<link rel="help" href="https://www.w3.org/TR/mediacapture-streams/#mediadevices">
+<meta name='assert' content='Check that the mediaDevices.enumerateDevices() method is present.'/>
+<meta name='flags' content='dom'/>
+<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
+</head>
+<body>
+<h1 class="instructions">Description</h1>
+<p class="instructions">This test checks for the presence of the
+<code>navigator.mediaDevices.enumerateDevices()</code> method.</p>
+<div id='log'></div>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script>
+test(function () {
+  assert_true(undefined !== navigator.mediaDevices.enumerateDevices(), "navigator.mediaDevices.enumerateDevices() exists");
+  }, "mediaDevices.enumerateDevices() is present on navigator");
```

Comment:
This is strangely described. It's `navigator.mediaDevices.enumerateDevices` that can be present, but what you're actually testing is that when invoked, it returns something other than undefined. Can you assert that it returns what the spec requires instead, i.e. a promise? AFAICT, the spec never rejects the promise, so you could even verify that it resolves to a `sequence<MediaDeviceInfo>`, even if empty.